docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,700 @@
+* [requestAsBrowser()](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions) function by specifying proxy configuration in the options.
+* [launchPuppeteer()](https://sdk.apify.com/docs/typedefs/launch-puppeteer#docsNav) by specifying the configuration in the function's options.
+
+The Apify SDK's [ProxyConfiguration](https://sdk.apify.com/docs/api/proxy-configuration) enables you to choose which proxies you use for all connections. You can inspect the current proxy's URL and other attributes using the [ProxyInfo](https://sdk.apify.com/docs/typedefs/proxy-info) property in your crawler's [page function](https://sdk.apify.com/docs/typedefs/cheerio-crawler-options#handlepagefunction).
+
+### [](#rotate-ip-addresses) Rotate IP addresses
+
+IP addresses are selected at random from all available proxy servers.
+
+```marked-tabs
+<marked-tab header="PuppeteerCrawler" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const requestList = await Apify.openRequestList(
+        "my-list", ["http://www.example.com"]
+    );
+    const proxyConfiguration = await Apify.createProxyConfiguration();
+
+    const crawler = new Apify.PuppeteerCrawler({
+        requestList,
+        proxyConfiguration,
+        handlePageFunction: async ({ page, request, proxyInfo }) => {
+            return Apify.pushData({ title: await page.title() });
+        },
+        handleFailedRequestFunction: ({ request }) => {
+            console.error("Request failed", request.url, request.errorMessages);
+        },
+    });
+
+    await crawler.run();
+});
+</marked-tab>
+
+
+<marked-tab header="requestAsBrowser()" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const proxyConfiguration = await Apify.createProxyConfiguration();
+    try {
+        const { body } = await Apify.utils.requestAsBrowser({
+            url: "https://www.example.com",
+            proxyUrl: proxyConfiguration.newUrl(),
+        });
+        console.log(body); // returns HTML of returned page
+    } catch (e) {
+        console.error(e);
+    }
+});
+</marked-tab>
+```
+
+### [](#single-ip-address-from-specific-groups) Single IP address from specific groups
+
+The IP address is selected from the `SHADER` and `BUYPROXIES94952` proxy groups.
+
+```marked-tabs
+<marked-tab header="PuppeteerCrawler" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const requestList = await Apify.openRequestList(
+        "my-list", ["http://www.example.com"]
+    );
+    const proxyConfiguration = await Apify.createProxyConfiguration({
+        // if you need to use more then one group
+        // simply add the additional ID to the array below
+        groups: ["SHADER", "BUYPROXIES94952"],
+    });
+
+    const crawler = new Apify.PuppeteerCrawler({
+        requestList,
+        proxyConfiguration,
```

Comment:
This config will give you a random IP for each page, not a single IP, because it does not use sessions anywhere.